web-truyen/src/components/content/DetailComic.jsx

```diff
@@ -0,0 +1,505 @@
+import BannerDetail from "../../images/banner-detail.png";
+import avatar from "../../images/avatar.jpg";
+import { ReactComponent as IconEye } from "../../images/icons/icon-eye.svg";
+import { Link } from 'react-router-dom'
+
+function DetailComic({ comic }) {
+  return (
+    <div className="detail-container">
+      <div className="detail-banner">
+        <img src={BannerDetail} alt="Banner Detail" />
+      </div>
+      <div className="detail-content">
+        <div className="detail-content__chapter">
+          <div className="detail-content__chapter-btn">
+            <Link
```

Comment:
handle action for 2 button

Reply:
https://github.com/diennn-gg/training-jun/assets/138417943/29b2fb33-f75b-4845-bcb7-5703dbd25aa8